catalogue/webapp/pages/search/works.tsx

```diff
@@ -96,11 +112,34 @@ export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
 
   return (
     <>
-      <h1 className="visually-hidden">Works Search Page</h1>
+      <Head>
+        {works?.prevPage && (
```

Comment:
We talked about how this is for Google spider-ing reasons and needs a bigger conversation; let's create a ticket to track those discussions.

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/8936

Did a bit more research and documented it in there